SPECS/golang/golang.spec

```diff
@@ -13,7 +13,7 @@
 %define __find_requires %{nil}
 Summary:        Go
 Name:           golang
-Version:        1.21.6
+Version:        1.22.4
```

Comment:
I don't think this would work due to bootstrapping issues

Reply:
As discussed yesterday I am using 1.4 -> 1.17 -> 1.21 -> 1.22 chain of bootstrapping to compile golang 1.22

Reply:
I am able to build the golang 1.22

```
INFO[0000][scheduler] Building: golang-1.22.4-1.cm2.src.rpm
INFO[0310][scheduler] Built: golang-1.22.4-1.cm2.src.rpm -> [/home/nerdywit/azurelinux/out/RPMS/x86_64/golang-1.22.4-1.cm2.x86_64.rpm]
INFO[0310][scheduler] 0 currently active build(s): [].
INFO[0310][scheduler] All packages built
INFO[0311][scheduler] ---------------------------
INFO[0311][scheduler] --------- Summary ---------
INFO[0311][scheduler] ---------------------------
INFO[0311][scheduler] Number of prebuilt SRPMs:           0
INFO[0311][scheduler] Number of prebuilt delta SRPMs:     0
INFO[0311][scheduler] Number of skipped SRPMs tests:      0
INFO[0311][scheduler] Number of built SRPMs:              1
INFO[0311][scheduler] Number of passed SRPMs tests:       0
INFO[0311][scheduler] Number of unresolved dependencies:  0
INFO[0311][scheduler] Number of blocked SRPMs:            0
INFO[0311][scheduler] Number of blocked SRPMs tests:      0
INFO[0311][scheduler] Number of failed SRPMs:             0
INFO[0311][scheduler] Number of failed SRPMs tests:       0
INFO[0311][scheduler] Number of toolchain RPM conflicts:  0
INFO[0311][scheduler] Number of toolchain SRPM conflicts: 0
INFO[0311][scheduler] Built SRPMs:
INFO[0311][scheduler] --> golang-1.22.4-1.cm2.src.rpm
```